provisioning/pythondotorg.yml

```diff
@@ -36,13 +31,6 @@
       changed_when:
         "result.stdout is defined and '0 upgraded, 0 newly installed, 0 to remove and' not in result.stdout"
 
-    - name: Install python-lxml deps
-      become: yes
-      apt: pkg=python-lxml state=build-dep
```

Comment:
Does this get installed by BeautifulSoup now?

Reply:
Sure I will remove # - acl in the next amendment.
Regading lxml:
I have done local tests yesterday during the Python Sprint of the new box and was able to launch the site with initial data.
Also built the docs from txt to html and run the management command to do the same.
It looks like lxml required works now without specifying those dependencies.
I will retest all of that though just to make sure. Is there anything else that lxml is used for?

Reply:
> Is there anything else that lxml is used for?

I think BeautilfulSoup + lxml is only used by the PEP converter, so running `./manage.py test peps` may help to confirm this.

Reply:
`lxml` is now available as a wheel, so `pip install` does not need to build it any-more which means we don't need the `pkg=python-lxml state=build-dep` step and we can also remove `libxml2-dev` and `libxslt-dev` from the `Install base packages` step.

I have tested locally, `./manage.py test peps` passes and I can generate the PEPs.

Reply:
Awesome, thanks for testing @Gautier! @Tomasz-Kluczkowski could you remove those packages as well?

Reply:
Working on it.

Reply:
Added Gautier to the commit as a co-autor - he helped a lot during the sprint.